section3/exercises/loops-arrays.js

```diff
@@ -18,22 +18,35 @@ for (var i = 0; i < animals.length; i++) {
 
 // YOU DO: Write code below that iterates through a list of animals and prints "The
 // <animal> is awesome!" for each animal.
+var animals = ['Zebra', 'Giraffe', 'Elephant']
 
+for (var i = 0; i < animals.length; i++) {
+  console.log('The ' + animals[i] + ' is amazing')
+}
 
 
 // YOU DO: Create an array of foods and then iterate over that array to log "Add
 // <food> to shopping list" for each food item.
+var food = ['tofu', 'broccoli', 'rice', 'soy sauce', 'peppers']
 
+for (var i = 0; i < food.length; i++){
+  console.log('Add ' + food[i] + ' to shopping list')
+}
 
 
 // YOU DO: Create an array of numbers and then iterate over that array to log double
 // of each of the number. (Ex: 4 doubled is 8.)
-
+var numbers= [1,2,3,4,5,6,7,8]
+for (var i = 1; i <= 8; i++) {
+  console.log(numbers.length *= 2)
+}
 
 // YOU DO: Using the array of names below, print out "Happy Birthday <name>" for
 // each name except the first two.
```

Comment:
I set var i = 2,; i < numbers.length; i++
 since the first two elements are index 0 & 1. I ran that in my terminal in it omitted the first two elements while leaving room to add additional elements as suggested in the comment above.